test/torchaudio_unittest/sox_effect/dataset_test.py

```diff
@@ -122,3 +123,36 @@ def test_apply_effects_tensor(self):
         )
         for batch in loader:
             assert batch.shape == (32, 2, 2 * sample_rate)
+
+
+def speed(path):
+    wav, sample_rate = torchaudio.backend.sox_io_backend.load(path)
+    effects = [
+        ['speed', '1.03756523535464655'],
+        ['rate', f'{sample_rate}'],
+    ]
+    return torchaudio.sox_effects.apply_effects_tensor(wav, sample_rate, effects)[0]
+
+
+@skipIfNoExtension
+class TestProcessPoolExecutor(TempDirMixin, PytorchTestCase):
+    backend = "sox_io"
+
+    def setUp(self):
+        sample_rate = 16000
+        self.flist = []
+        for i in range(10):
+            path = self.get_temp_path(f'{i}.wav')
+            data = get_whitenoise(n_channels=1, sample_rate=sample_rate, duration=1, dtype='float')
+            save_wav(path, data, sample_rate)
+            self.flist.append(path)
+
+    def test_executor(self):
+        """Test that apply_effects_tensor with speed + rate does not crush
+
+        https://github.com/pytorch/audio/issues/1021
+        """
+        executor = ProcessPoolExecutor(1)
```

Comment:
Why do you do this?

Reply:
This is the reproduction of tests reported in https://github.com/pytorch/audio/issues/1021